### PR TITLE
added databases week two homework

### PR DESCRIPTION
**Pull request by STEP0HEN**

### CODE REVIEW

File: databases/week2/databases-week2-Homework/school_databse.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `class`(`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- create index on name column from student table
+CREATE INDEX student_name_index
+ON student (name);
+
+-- Add column to table
+
+ALTER TABLE class ADD COLUMN status ENUM('not started', 'ongoing', 'finished')
```
Comment (babakbandpey) [inline databases/week2/databases-week2-Homework/school_databse.sql:35]: What would happen when the student has more than one class?
The answer is that you need a table that connects the student to classes.
maybe a student_classes table with studen_id and class_id as foreign keys.

File: databases/week2/databases-week2-Homework/more_queries.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    SELECT id 
+    FROM user
+    WHERE name = 'Donald Duck'
+)
+AND s.id = (
+    SELECT id
+    FROM status
+    WHERE name = 'Not started'
+);
+
```
Comment (babakbandpey) [inline databases/week2/databases-week2-Homework/more_queries.sql:27]: They query fails if there are more than two whose names are Donald Duck.
Can you come up with a solution to this?

**Review by babakbandpey — CHANGES_REQUESTED** [review]
Hello STEP0HEN,

I have some challenges for you. I see that you already are very good at doing queries, so I did not provide the answer and the answer is simple.
I look forward to see how you fix it.
Thank you.

**Comment by STEP0HEN** [comment]
Hi Babak Thank you for the review, I implemented the changes.

File: databases/week2/databases-week2-Homework/school_databse.sql (hunk 1 above)
Comment (STEP0HEN) [reply databases/week2/databases-week2-Homework/school_databse.sql:35]: ok

File: databases/week2/databases-week2-Homework/more_queries.sql (hunk 2 above)
Comment (STEP0HEN) [reply databases/week2/databases-week2-Homework/more_queries.sql:27]: I'm thinking it's the equal sign (=), so I changed it to 'LIKE' instead.

**Pull request merged**